packages/fiori-elements-writer/src/data/defaults.ts

```diff
@@ -80,5 +80,12 @@ export function setAppDefaults<T>(feApp: FioriElementsApp<T>): FioriElementsApp<
         (feApp.service as OdataService).model = defaultModelName;
     }
 
+    // minimum UI5 version depending on the template required
+    feApp.ui5 = feApp.ui5 ?? {};
```

Comment:
Why ?? and not || ?

Reply:
It doesn't make a difference here but I prefer it with strings to accept an empty string as input therefore I use it everywhere

Reply:
interesting !